addons/im_livechat/static/tests/embed/feedback_panel_tests.js

```diff
@@ -40,7 +40,7 @@ QUnit.test("Close without feedback", async (assert) => {
     await contains(".o-mail-Message-content", { text: "Hello World!" });
     await click("[title='Close Chat Window']");
     await click("button", { text: "Close conversation" });
-    await contains(".o-livechat-LivechatButton", { count: 0 });
+    await contains(".o-livechat-LivechatButton");
```

Comment:
I assume it was a bug that `LivechatButton` was not shown and made the test pass?
Not a huge bug regardless, but just wondering this is the reason behind this diff

Reply:
Live chat button should be shown after closing the thread. Somehow the test was passing before (while it worked in real life^^). Anyway the correct behavior is the one asserted after this diff.